resources/packaging/bluebanquise.spec

```diff
@@ -103,26 +86,11 @@ cp -a tools/bluebanquise-playbook %{buildroot}%{_sbindir}/
 %dir %{_sysconfdir}/%{name}/
 %config(noreplace) %{_sysconfdir}/%{name}/ansible.cfg
 %config %{_sysconfdir}/%{name}/internal/
-%{_sysconfdir}/%{name}/roles/customs/
+%{_sysconfdir}/%{name}/roles/custom/
 %attr(750,root,root) %{_sbindir}/bluebanquise-playbook
 
-
-# Create subpackages for each addon role
-%{lua:
-local name = rpm.expand("%{name}")
-local version = rpm.expand("%{version}")
-
-for role in string.gmatch(rpm.expand("%{roles_addons}"), "[%w_-]+")
-do
-  print("%package addons-" .. role .. "\n")
-  print("Summary: Addon " .. role .. " for BlueBanquise\n")
-  print("Requires: " .. name .. " == " .. version .. "\n")
-  print("%description addons-" .. role .. "\n")
-  print("%files addons-" .. role .. " -f rolesfiles.addons." .. role .. "\n")
-end}
-
-
 %changelog
+* Thu Feb 25 2021 Benoit Leveugle <benoit.leveugle@gmail.com>
 * Sun Feb  2 2020 Bruno Travouillon <devel@travouillon.fr>
 * Tue Apr 14 2020 strus38 <indigoping4cgmi@gmail.com>
```

Comment:
The changelog is unordered ?!? Your change is correct, but last line is not. Could you use this PR to reorder the changelog?